gradoop-flink/src/main/java/org/gradoop/flink/datagen/transactions/foodbroker/config/FoodBrokerConfig.java

```diff
@@ -98,7 +89,8 @@ public static FoodBrokerConfig fromJSONString(String configString, String resour
   public List<String> getStringValuesFromFile(String fileName) {
     List<String> values = null;
     try {
-      values = FileUtils.readLines(FileUtils.getFile(resourcePath + "/" + fileName));
+      values = FileUtils.readLines(FileUtils.getFile(
+        FoodBrokerConfig.class.getResource("/foodbroker/" + fileName).getFile()));
```

Comment:
I think this will not work if gradoop tries to access the resource files when packed in a jar. What about 
```java
org.apache.commons.io.IOUtils.toString(this.getClass().getResourceAsStream(fileName),"UTF-8");
```
 instead? (plus closing the inputstream)